CHANGELOG-developer.next.asciidoc

```diff
@@ -231,6 +231,7 @@ client will aggregate metrics by input on this registry. It'll collect
 unregister input metrics without relaying on the global 'dataset' namespace.{pull}42618[42618] {issue}42761[42761]
 - Added exponential backoff retry logic for vSphere client logout operation in metricbeat. {issue}43449[43449]
 - Improved Filebeat Websocket input exponential backoff logic to produce a smoother backoff curve. {pull}44069[44069]
+- Added `autoops_es` module to x-pack/metricbeat. {pull}44565[44565]
```

Comment:
You want this in CHANGELOG.next.asciidoc. That is the user facing changelog that ends up in the official release notes (assuming you want it to appear there).